plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/StandardColumnMappings.java

```diff
@@ -497,6 +545,16 @@ public static long toPrestoTimestamp(TimestampType timestampType, LocalDateTime
         return instant.getEpochSecond() * MICROSECONDS_PER_SECOND + roundDiv(instant.getNano(), NANOSECONDS_PER_MICROSECOND);
     }
 
+    public static LongTimestamp toLongTimestamp(TimestampType timestampType, LocalDateTime localDateTime)
+    {
+        long precision = timestampType.getPrecision();
+        checkArgument(precision > MAX_SHORT_PRECISION, "Precision is out of range: %s", precision);
```

Comment:
this check is not needed, but im ok keeping it

Reply:
I agree, but I wanted to be consistent with `toPrestoTimestamp`